test/fork/migration-v2.js

```diff
@@ -689,143 +683,309 @@ describe('V2 upgrade', function () {
+    const stakingPoolCount = await this.stakingPoolFactory.stakingPoolCount();
+    expect(stakingPoolCount).to.be.equal(topStakers.length + 1); // +1 because Armor has 2 pools
+
+    // Check the new staking pools have the correct deposits and stakers have the correct balances
+    const depositsInStakingPools = {};
+    for (let i = 0; i < stakingPoolCount; i++) {
+      const { deposits } = await this.tokenController.stakingPoolNXMBalances(i);
+      console.log(`Staking pool ${i} deposit: ${deposits.toString()}`);
+      depositsInStakingPools[i] = deposits;
+    }
 
-    expect(nxmBalancesAfter[NEXUSMUTUAL_FOUNDATION].sub(nxmBalancesBefore[NEXUSMUTUAL_FOUNDATION])).to.be.equal(
-      depositAmounts[NEXUSMUTUAL_FOUNDATION],
+    console.log('Checking the new staking pools have the correct deposits and stakers have the correct balances');
+
+    // Check Armor - poolId = 0
+    // 5% of the stake is unlocked
+    // 71.25% of the stake moves to AAA Pool (95% * 75% of the stake)
+    // 23.75% of the stake moves to AA Pool (95% * 25% os the stake)
+
+    // TODO: We can avoid the precision tolerance here if we handle things a bit differently in Solidity
+    let precisionTolerance;
+
+    // Nexus Mutual Foundation Pool
+    console.log('Nexus Mutual Foundation Pool');
+    const foundationPoolId = 0;
+    // The entire NXM balance is unlocked and sent back to the Foundation
+    // TODO: Needs some Solidity changes to be able to fully migrate them as well
+    expect(depositsInStakingPools[foundationPoolId]).to.be.equal(0); // depositInPS[FOUNDATION]
+    expect(nxmBalancesAfter[FOUNDATION]).to.be.equal(nxmBalancesBefore[FOUNDATION].add(depositInPS[FOUNDATION]));
+
+    // Hugh Pool
+    console.log('Hugh Pool');
+    const hughPoolId = 1;
+    precisionTolerance = 1; // 1 wei
+    expect(depositsInStakingPools[hughPoolId]).to.be.equal(depositInPS[HUGH].sub(precisionTolerance));
+    // No NXM gets unlocked, so the balance shouldn't change
+    precisionTolerance = 1; // 1 wei
+    expect(nxmBalancesBefore[HUGH]).to.be.equal(nxmBalancesAfter[HUGH].sub(precisionTolerance));
+
+    // Armor AAA Pool
+    console.log('Armor AAA Pool');
+    const armorAAAPoolId = 2;
+    // 5% of the AAA allocation must be unlocked
+    const expectedArmorAAAPoolBalance = depositInPS[ARMOR].mul(75).div(100).mul(95).div(100);
+    precisionTolerance = 2;
+    expect(depositsInStakingPools[armorAAAPoolId]).to.be.equal(expectedArmorAAAPoolBalance.sub(precisionTolerance));
+
+    // Armor AA Pool
+    console.log('Armor AA Pool');
+    const armorAAPoolId = 3;
+    // 5% of the AA allocation must be unlocked
+    const expectedArmorAAPoolBalance = depositInPS[ARMOR].mul(25).div(100).mul(95).div(100);
+    precisionTolerance = 2;
+    expect(depositsInStakingPools[armorAAPoolId]).to.be.equal(expectedArmorAAPoolBalance.sub(precisionTolerance));
+
+    // Overall we must unlock 5% of Armor's total tokens in PS
+    precisionTolerance = 6;
```

Comment:
a bump here by 2 for precision Tolerance. 

As discussed your fix should address these + any other thing that's necessary.